### DISCUSSION TITLE
Unable to invite members after rolling back to the free tier.

### DISCUSSION THREAD

**Discussion by paulmenefee** [community/community#38610] [New to GitHub]
I started an organization on the middle tier and added two users, great.  That went well, then after discussing the project with the team we decided that the middle tier wasn't needed so I reverted it back to free tier but now we are unable to invite users until the sub for the paid tier expires in a year! Why do I need to wait a year?  And more importantly how can I add users on the free tier now?

**Comment by queenofcorgis** [comment]
Hi there! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :) Thanks!
